src/Module/TwoFactor/Verify.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Friendica\Module\TwoFactor;
+
+use Friendica\BaseModule;
+use Friendica\Core\L10n;
+use Friendica\Core\PConfig;
+use Friendica\Core\Renderer;
+use Friendica\Core\Session;
+use PragmaRX\Google2FA\Google2FA;
+
+/**
+ * Page 1: Authenticator code verification
```

Comment:
Do we have any possibility to "sort" the pages per name too like
`src/Module/TwoFactor/01_Verify.php`
It seems for me the order is important.

Reply:
Nah, the filename must match the class name, and the class name can't start with a number.

These page numbers were only for me during the design phase where I laid out all the necessary pages in plain text before creating the module classes. I don't think they are useful beyond these comments.